internal/venice-common/src/main/proto/VeniceReadService.proto

```diff
@@ -37,6 +37,7 @@ message CountByValueRequest {
   string resourceName = 2;
   int32 topK = 3;
```

Comment:
Seems we did not need to pass topK to server, as we always need to return the full mapping from value to count.